bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -113,16 +114,35 @@ def test_import_export_workflow_from_local_path(self):
             shutil.rmtree(export_dir)
         self.assertIsInstance(exported_wf_dict, dict)
 
+    def test_import_publish_export_workflow_from_local_path(self):
```

Comment:
Can you remove `export_` from the method name?

Reply:
Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/7e2a16b2bff1d56d44311c88bf761f273e94ddb0